### ISSUE TITLE
Question to beacon behavior

### ISSUE THREAD

**Issue by pittbull** [andrewjfreyer/monitor#230]
In the documentation you are writing:

> The first floor constantly monitors for beacons (-b) advertisements and anonymous advertisements, which may be sent by our phones listed in the known_static_addresses file. In response to a new anonymous advertisement, monitor will initiate an ARRIVAL scan for whichever of our phones is not present. If one of those devices is seen, an mqtt message is sent to Home Assistant reporting that the scanned phone is "home" with a confidence of 100%.

I am using this setup for my garage computer (-b -td) and I am experiencing an mqtt message being sent for every beacon it detects. Is this correct behavior? From your text I assumed that if it detects a becaon it initiates an arrival scan, and if any if the know_statics are detected _then_ it will send an mqtt message.

I am using -b to make sure the garage computer detects me as soon as possible, and being a somewhat busy are I pick up a fair amount of beacons - all of which are reported via mqtt. Is it possible to limit these mqtt messages?

**Comment by andrewjfreyer** [comment]
Yes, this is correct behavior. Are you trying to detect a beacon for your arrival?

**Comment by pittbull** [comment]
My garage bluetooth, or rather my phone, does not seem to penetrate the wall of my garage, hence my computer is not able to detect me arriving. However if a scan is triggered my computer is able to detect my phone on the outside.

Basically I want monitor to respond to all bluetooth activity to initiate a scan and thereby finding my phone.
